client/src/style/index.pcss

```diff
@@ -8,10 +8,22 @@
   & .squids-list__tile-header {
     @apply font-medium;
   }
+  & .squids-list__tile {
+    @apply m-auto my-3 w-7/12 text-lg text-center bg-zinc-300 text-cyan-700 p-3 rounded-xl shadow-md;
```

Comment:
A grid might be more appropriate here. setting the width this way may be unreliable as browser widths vary.